client/post-editor/editor-sidebar/index.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EditorDrawer from 'post-editor/editor-drawer';
+import EditorGroundControl from 'post-editor/editor-ground-control';
+import DraftList from 'my-sites/drafts/draft-list';
+import EditorSidebarHeader from './header';
+import EditorActionBar from 'post-editor/editor-action-bar';
+
+export default React.createClass( {
+	displayName: 'EditorSidebar',
+
+	propTypes: {
+		allPostsUrl: PropTypes.string,
+		sites: PropTypes.object,
+		onTitleClick: PropTypes.func,
+		savedPost: PropTypes.object,
+		post: PropTypes.object,
+		isNew: PropTypes.bool,
+		isDirty: PropTypes.bool,
+		isSaveBlocked: PropTypes.bool,
+		hasContent: PropTypes.bool,
+		isSaving: PropTypes.bool,
+		isPublishing: PropTypes.bool,
+		onSave: PropTypes.func,
+		onPreview: PropTypes.func,
+		onPublish: PropTypes.func,
+		onTrashingPost: PropTypes.func,
+		site: PropTypes.object,
+		type: PropTypes.string,
+		showDrafts: PropTypes.bool
+	},
+
+	render() {
+		return (
+			<div className="post-editor__sidebar">
+				<EditorSidebarHeader
+					allPostsUrl={ this.props.allPostsUrl } />
+				{ this.props.showDrafts
+					? <DraftList
+						sites={ this.props.sites }
+						onTitleClick={ this.props.onTitleClick }
+						showAllActionsMenu={ false }
+						siteID={ this.props.site ? this.props.site.ID : null }
+						selectedId={ this.props.post ? this.props.post.ID : null }
+					/>
+					: <div>
```

Comment:
Indentation feels a bit odd here. A few ideas to consider:

- Instead of a `<div>` wrapper, you can render an array of elements. If you do this, you'll also need to add a key to each
 - e.g. `[ <EditorActionBar key="action-bar" />, <EditorGroundControl key="ground-control " />, /* ... */ ]`
- Split render paths into individual functions
 - e.g. `this.props.showDrafts ? this.renderDraftsList() : this.renderSidebar()`
- Compose content into a variable
 - e.g. 

```js
let content;
if ( this.props.showDrafts ) {
	content = /* ... */;
} else {
	content = /* ... */;
}

return (
	<div className="post-editor__sidebar">
		 { content }
	</div>
);
```